.github/workflows/build_and_test.yml

```diff
@@ -48,6 +48,8 @@ jobs:
             -Pdetekt.multiplatform.disabled=true
             -PgprUser=${{ github.actor }}
             -PgprKey=${{ secrets.GITHUB_TOKEN }}
+            -Pgpr.user=${{ github.actor }}
```

Comment:
maybe fix configuration of maven repository to use `grpUser` instead of `gpr.user`

Reply:
It is considered that `gpr.user` is better than `gprUser` (there is no actual difference, but the first variant is better for yml)
Should ask Andrey K if it is ok to change all the `gprUser` to `gpr.user`

Reply:
Removed redundant props